src/nccl_ofi_interface_neuron.c

```diff
@@ -7,6 +7,12 @@
 #include "nccl_ofi.h"
 #include "nccl_ofi_api.h"
 
+static ncclResult_t init_v4(ncclDebugLogger_t logFunction)
+{
+	v4_neuron_interface = true;
```

Comment:
I was thinking of something way simpler here.  Instead of thread all this through to the platform file, why not just do this:

```
/* explain why we're doing this */
setenv("NCCL_OFI_PROTOCOL", "SENDRECV", 0);
```

Reply:
it would be even better if we had a setter function for the params, but that's for another day.

Reply:
Simplified patch according to comment.